src/bootstrap/src/utils/change_tracker.rs

```diff
@@ -86,4 +86,9 @@ pub const CONFIG_CHANGE_HISTORY: &[ChangeInfo] = &[
         severity: ChangeSeverity::Info,
         summary: "Use of the `if-available` value for `download-ci-llvm` is deprecated; prefer using the new `if-unchanged` value.",
     },
+    ChangeInfo {
+        change_id: 118703,
+        severity: ChangeSeverity::Warning,
+        summary: "Removed rust.run_dsymutil config option, as it was unused.",
+    },
```

Comment:
Ok, changed.

Reply:
Done. I wonder if we could remove `change-id` from the example TOML, and instead do something like this: If we don't find `change-id` in `config.toml`, add it there, and autofill it with the latest change record. So that we don't have to continuously change this number in `config.example.toml`.

Reply:
> Done. I wonder if we could remove `change-id` from the example TOML, and instead do something like this: If we don't find `change-id` in `config.toml`, add it there (like, actually edit the TOML file, not just use a default value in memory, that wouldn't help the detection), and autofill it with the latest change record. So that we don't have to continuously change this number in `config.example.toml`.

I am okay with removing that from the example configuration.

Reply:
Ok, I'll try to do that in a different PR, I'll take a look at how difficult it would be to actually modify the TOML file.